tests/test_utils.py

```diff
@@ -24,3 +22,16 @@ def test_check_paths_exists(test_args):
     # path is not a valid type
     setattr(test_args, "output_data", 123)
     assert check_paths_exists(test_args) == False
+
+def test_read_config_file(good_config):
+
+    dict = read_config_file(good_config)
+    assert dict['code'] == 'code'
+    assert dict['input_data'] == 'input_data'
+    assert dict['csv'] is None
+
+def test_dictionary_printer(capsys):
+    dict = {'a': 'hi', 'b' :'bye'}
+    dictionary_printer(dict)
+    captured = capsys.readouterr()
+    assert 'a: hi\nb: bye' in captured.out
```

Comment:
I get an error running this test - the test works if I remove the spaces from the expected output so that I have `'a:hi\nb:bye'`.
@kevinxufs does this test work for you as is?

Reply:
Hi radka. This test seems to work for me. This test was recently modified as Eric suggested that I reformat the `dictionary_printer` function in `utils.py`. Could you check your `utils.py` to see if there is a space in between the curly brackets?